### ISSUE TITLE
missions created in DCS 2.7+ occasionally render AN/ASN-41 navigation computer inoperable

### ISSUE THREAD

**Issue by callmepartario** [heclak/community-a4e-c#344]
**Describe the bug**
Newly created missions in 2.7's mission editor render the navigation computer inoperable. 

If you load a mission created in 2.6, loading it in 2.7 has everything working just fine. 

Additionally re-saving it to 2.7's mission format (including making cloud changes) does not appear to cause this breakage (in the develop branch, the quick-start missions are all updated and are not showing this behavior).

Thanks to SPINEG, HawkMoon and others in the discord for helping reproduce a few things here. Any additional testing or reproduction would still be welcome!

**Software Information (please complete the following information):**
 - DCS Version: [e.g. DCS 2.7 Open Beta]
 - A-4E Version [e.g. v2.0.0 beta 3 (could be happening on older versions, have not tested)]

**Comment by callmepartario** [comment]
```
2021-04-29 14:04:56.764 ERROR   COCKPITBASE: NAV::update:[string "C:\Users\gargo\Saved Games\DCS.openbeta\Mods/aircraft/A-4E-C/Cockpit/Scripts/Systems/mission.lua"]:206: attempt to get length of global 'miz_carriers' (a nil value)
2021-04-29 14:04:52.178 ERROR   Lua::Config: Call error post_initialize:[string "C:\Users\gargo\Saved Games\DCS.openbeta\Mods/aircraft/A-4E-C/Cockpit/Scripts/Systems/mission.lua"]:104: attempt to index field 'action' (a nil value)
stack traceback:
    [C]: ?
    [string "C:\Users\gargo\Saved Games\DCS.openbeta\Mods/aircraft/A-4E-C/Cockpit/Scripts/Systems/mission.lua"]:104: in function 'decode_mission'
    [string "C:\Users\gargo\Saved Games\DCS.openbeta\Mods/aircraft/A-4E-C/Cockpit/Scripts/Nav/nav.lua"]:402: in function <[string "C:\Users\gargo\Saved Games\DCS.openbeta\Mods/aircraft/A-4E-C/Cockpit/Scripts/Nav/nav.lua"]:314>.
```

[dcs.log](https://github.com/heclak/community-a4e-c/files/6399353/dcs.log)

**Comment by callmepartario** [comment]
having trouble reproducing, a mission file would be most helpful, we've let hawkmoon know

**Comment by Sr-wolf78** [comment]
Hi 

I have several mission with this issue. How Can I share it? 

Thanks

**Comment by callmepartario** [comment]
the issue tracker can accept attachments -- just drag and drop or paste them in.

**Comment by Sr-wolf78** [comment]
> the issue tracker can accept attachments -- just drag and drop or paste them in.

Thanks callmepartario, please would you tell me where is the issue tracker ?  I am trying to attach the miz file here

**Comment by callmepartario** [comment]
you are posting on the issue tracker right now. you're there. just drag the mission file into these posts you're making.

**Comment by Sr-wolf78** [comment]
[La guarida del Lobo A4E.zip](https://github.com/heclak/community-a4e-c/files/6533379/La.guarida.del.Lobo.A4E.zip)

**Comment by Sr-wolf78** [comment]
for example. 

In my case, the navigation system is inactive and cannot be programmed either. You can hear the adjustment "clips" but the indicators remain at 0 and the system is inoperative. 

It only works editing missions already created in 2.6

If you need more missions, information, etc ... about this error, I will be happy to share them with everyone. 

thanks for your effort

**Comment by callmepartario** [comment]
fixed in 2.1.1!
